src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/yflows/YFlowRerouteSpec.groovy

```diff
@@ -300,10 +300,8 @@ class YFlowRerouteSpec extends HealthCheckSpecification {
 
         and: "Switch off all ports on the terminal switch of not intersected ISLs"
         Switch terminalSwitch = notIntersectedIsls.last().dstSwitch
-        List<PortDto> portsDown = []
-        topology.getRelatedIsls(terminalSwitch).each {
-            portsDown.add(antiflap.portDown(terminalSwitch.dpId, it.srcSwitch == terminalSwitch ? it.srcPort : it.dstPort))
-        }
+        def broughtDownIsls = topology.getRelatedIsls(terminalSwitch)
+        islHelper.breakIsls(broughtDownIsls)
```

Comment:
It seems like an old logic has been changed as a terminal switch can be src or dst for ISL, but we turn OFF src.port in the breakIsl(Isl isltoBreak) method.

Reply:
In this particular tests it doesn't matter. I've changed step description to reflect breaking ISLs on the terminal switch.